cg/meta/backup/backup.py

```diff
@@ -422,7 +418,11 @@ def remove_archived_spring_file(self, spring_file_path: Path) -> None:
 
     def is_spring_file_archived(self, spring_file_path: Path) -> bool:
         """Checks if a spring file is marked as archived in Housekeeper."""
-        return self.hk_api.files(path=str(spring_file_path)).first().to_archive
+        file = self.hk_api.files(path=str(spring_file_path)).first()
+        if file and not spring_file_path.exists():
```

Comment:
This check behaves differently than the original one. Realized we might need the original function.
Also, maybe the log message should be outside the if statement.